### ISSUE TITLE
Cmdlet checks on ".db" file's extension

### ISSUE THREAD

**Issue by kolmidad** [jdhitsolutions/MySQLite#17]
Hi,

There's a check that runs at cmdlet starting on database file extension.
It seems to allow ".db" pattern only and won't run in any other case.
We use ".sqlite" extension for naming our Sqlite database files as it is allowed (also default in Sqlite DB Browser).
This is handy for us because we may use several DB engines on the same filesystem with no ambiguity.

Will you agree to modify these cmdlets in a more permisive way ?
Is there any strong reason not to ?

Thanks a lot by advance as doing this allows us to add your work in our team's tool box.

**Comment by jdhitsolutions** [comment]
This is a reasonable request.  I can add a test for all the valid extensions. It looks like `Export-MysqliteDB` is the only command where this is an issue. Or is there some other command that is problematic?

**Comment by kolmidad** [comment]
yes indeed, it looks like other cmdlet accepts .sqlite extension
sorry for that I didn't checked far enough before asking
anyway I'm glad to read you'll extend the filename test
again thank you very much

**Comment by jdhitsolutions** [comment]
I've pushed v0.11.0 to the PowerShell Gallery with the new changes.
